src/components/screens/forms/form-reply-screen.tsx

```diff
@@ -106,24 +106,23 @@ const ReplyScreen: FC = () => {
     metaform.sections?.forEach(section => {
       section.fields?.forEach(field => {
         if (field.type === MetaformFieldType.Files) {
-          let value = getFieldValue(field.name as string);
-          if (!value) {
-            value = { files: [] };
+          const fileValue = getFieldValue(field.name as string) as FileFieldValue;
+          if (fileValue && fileValue.files) {
```

Comment:
FileFieldValue.files is typed as `export type FileFieldValue = {
  files: FileFieldValueItem[]
}`.
Empty array evaluates to true and therefore this condition evaluates to true even if there are no files. Is that intended? If not, `fileValue.files.length` would be more appropriate.